src/cpu/ukernel/brgemm.cpp

```diff
@@ -0,0 +1,157 @@
+/*******************************************************************************
+* Copyright 2025 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "oneapi/dnnl/dnnl_ukernel.h"
+
+#include "cpu/ukernel/c_types_map.hpp"
+
+#if DNNL_X64
+#include "cpu/x64/ukernel/brgemm.hpp"
+#endif
+
+#ifdef DNNL_EXPERIMENTAL_UKERNEL
+
+using namespace dnnl::impl;
+using namespace dnnl::impl::cpu;
+using namespace dnnl::impl::cpu::ukernel;
+
+status_t dnnl_brgemm_create(brgemm_t **brgemm, dim_t M, dim_t N, dim_t K,
+        dim_t batch_size, dim_t lda, dim_t ldb, dim_t ldc, data_type_t a_dt,
+        data_type_t b_dt, data_type_t c_dt) {
+#if DNNL_X64
+    return x64::ukernel::dnnl_brgemm_create(
+            brgemm, M, N, K, batch_size, lda, ldb, ldc, a_dt, b_dt, c_dt);
+#endif
+    return status::success;
```

Comment:
```suggestion
    return status::unimplemented;
```

Why return success and not unimplemented?
Also, we have to think if we will ever add reference path or not to cover backend specific limitations.